src/components/Nav.js

```diff
@@ -1,8 +1,9 @@
 /**
  * Nav component
  * This is the menu will allow users to access different pages
- * such as documentation and getting started 
- * and the search bar provide a quick search for files 
+ * such as documentation and getting started
+ * and the search bar provide a quick search for files
+
```

Comment:
1) What do you mean all the new assets?

2) We need the image-loader because the bg-header img is referencered is in the videoComponent.scss file and will be dynamically rendered everytime we build.